### PR TITLE
Update test.cpp

### PR DESCRIPTION
**Pull request by qaz5823091**

### CODE REVIEW

**Review by tianyo1110 — APPROVED** [review]
OK Fine !

File: test.cpp [hunk 1]
```diff
@@ -3,5 +3,5 @@ using namespace std;
 
 int main()
 {
-  cout << "Hellow, orld!" << endl;
+  cout << "Hello, worlb!" << endl;
```
Comment (Ericostco) [inline test.cpp:6]: Should be "Hello world"?

**Review by Ericostco — COMMENTED** [review]
Should be "Hello world"?

**Review by Ericostco — CHANGES_REQUESTED** [review]
Should be "Hello world"?

**Review by Ericostco — APPROVED** [review]
looks great!

**Pull request merged**